docs/index.md

```diff
@@ -5,49 +5,4 @@ feature_maturity: stable
 enterprise: 'no'
 ---
 
-Apache Spark is a fast and general-purpose cluster computing system for big data. It provides high-level APIs in Scala, Java, Python, and R, and an optimized engine that supports general computation graphs for data analysis. It also supports a rich set of higher-level tools including Spark SQL for SQL and DataFrames, MLlib for machine learning, GraphX for graph processing, and Spark Streaming for stream processing. For more information, see the [Apache Spark documentation][1].
```

Comment:
Can you instead modify your/suzanne's docs copying script to copy this source into that location?  I think all the documentation source should live with the project, otherwise it's difficult to change as the product changes.